### PR TITLE
Add HTTP/2 input buffer size config

### PR DESCRIPTION
**Pull request by nezihyigitbasi**
This is a useful knob to expose.

### CODE REVIEW

**Comment by nezihyigitbasi** [comment]
@electrum or @dain can you please take a look? Seems like I don't have assign rights in this repo.

File: http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java [hunk 1]
```diff
@@ -40,15 +43,14 @@
     public static final String JAVAX_NET_SSL_TRUST_STORE = "javax.net.ssl.trustStore";
     public static final String JAVAX_NET_SSL_TRUST_STORE_PASSWORD = "javax.net.ssl.trustStorePassword";
 
-    private boolean http2Enabled;
-    private Duration connectTimeout = new Duration(1, TimeUnit.SECONDS);
+    private Duration connectTimeout = new Duration(1, SECONDS);
```
Comment (electrum) [inline http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java]: These unrelated changes make it hard to read, can you pull this into a separate commit?

Comment (electrum) [inline http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java]: Nevermind, I did that. Will run tests and merge

**Comment by electrum** [comment]
Merged, thanks!
